### PR TITLE
[PPML]Update the document of graphene

### PR DESCRIPTION
**Pull request by Forest216**
* Refine Graphene K8S doc
* Add SGX memory configuration image

### CODE REVIEW

File: ppml/trusted-big-data-ml/python/docker-graphene/README.md [hunk 1]
```diff
@@ -573,7 +573,7 @@ kubectl create secret generic spark-secret --from-literal secret=YOUR_SECRET
 **The secret created (`YOUR_SECRET`) should be the same as the password you specified in section 1.1**
 
 ### 1.3 Start the client container
-Configure the environment variables in the following script before running it.
+Configure the environment variables in the following script before running it. See [Bigdl ppml SGX related configurations](#1-bigdl-ppml-sgx-related-configurations) for details of memory configurations.
```
Comment (qiyuangong) [inline ppml/trusted-big-data-ml/python/docker-graphene/README.md:576]: See change to Check

**Review by qiyuangong — APPROVED** [review]
LGTM

File: ppml/trusted-big-data-ml/python/docker-graphene/README.md [hunk 2]
```diff
@@ -699,6 +699,9 @@ You can run your own Spark Appliction after changing `--class` and jar path.
 
 ### Configuration Explainations
 #### 1. Bigdl ppml SGX related configurations
+
+<img title="" src="../../../../docs/readthedocs/image/ppml_memory_config.png" alt="ppml_memory_config.png" data-align="center">
+
 The following parameters enable spark executor running on SGX.
```
Comment (glorysdj) [inline ppml/trusted-big-data-ml/python/docker-graphene/README.md:705]: add more explanations for mem settings

**Review by glorysdj — APPROVED** [review]
LGTM

**Pull request merged**